frontend/src/app/modules/auth/register-form/register-form.component.scss

```diff
@@ -21,20 +21,21 @@
 }
 
 .registration-form {
-    margin-top: 40px;
+    margin-top: 5%;
     width: 360px;
 
     .input-label {
         margin: 0;
-        margin-bottom: 2%;
         font-size: 13px;
         font-weight: 600;
         color: #2c2c2c;
     }
 
     .input-form {
         width: 100%;
-
+        ::ng-deep .mat-form-field-flex {
```

Comment:
Why do you need `::ng-deep` here?

Reply:
i did not find another way to change size of mat-field element